server/src/managers/client_presence_manager.rs

```diff
@@ -159,15 +159,22 @@ impl<T: DisplacedPresenceListener> ClientPresenceManager<T> {
         Ok(presence_keys.len() as u8)
     }
 
-    fn is_locally_present(&self, presence_key: &str) -> bool {
-        self.displacement_listeners.contains_key(presence_key)
+    pub fn is_locally_present(&self, protocol_address: &ProtocolAddress) -> bool {
```

Comment:
Det er navngivning vi bruger andre steder:

```suggestion
    pub fn is_locally_present(&self, address: &ProtocolAddress) -> bool {
```